tasks/libs/releasing/version.py

```diff
@@ -239,40 +241,28 @@ def _get_highest_repo_version(
     return highest_version
 
 
-def _get_release_version_from_release_json(release_json, version_re, release_json_key=None):
+def _get_release_version_from_release_json(release_json, version_re, release_json_key):
     """
-    If release_json_key is None, returns the highest version entry in release.json.
-    If release_json_key is set, returns the entry for release_json_key of the highest version entry in release.json.
+    returns the release component version for release_json_key in the dependencies entry of the release.json
     """
 
-    release_version = None
     release_component_version = None
+    dependencies_entry = release_json.get(RELEASE_JSON_DEPENDENCIES, None)
 
-    # Get the release entry for the given Agent major version
-    release_entry_name = "release"
-    release_json_entry = release_json.get(release_entry_name, None)
-
-    # Check that the release entry exists, otherwise fail
-    if release_json_entry:
-        release_version = release_entry_name
-
-        # Check that the component's version is defined in the release entry
-        if release_json_key is not None:
-            match = version_re.match(release_json_entry.get(release_json_key, ""))
-            if match:
-                release_component_version = _create_version_from_match(match)
-            else:
-                print(
-                    f"{release_entry_name} does not have a valid {release_json_key} ({release_json_entry.get(release_json_key, '')}), ignoring"
-                )
-
-    if not release_version:
-        raise Exit(f"Couldn't find any matching {release_version} version.", 1)
+    if not dependencies_entry:
```

Comment:
```suggestion
    if dependencies_entry is None:
```
As you explicitly returns `None` from the `dict.get` method when this key is not found you should test if it is `None`